lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -121,6 +121,7 @@ sub list_running_ajax {
         state => [OpenQA::Jobs::Constants::EXECUTION_STATES],
         match => $self->get_match_param,
         groupid => $self->param('groupid'),
+        limit => ($self->param('limit') // undef),    # show all running by default
```

Comment:
I don't think this will work because if it is `undef` you have the fallback to `OpenQA::App->singleton->config->{global}->{query_limit}` in `prepare_complex_query_search_args`.